std/js/src/main/scala/cats/effect/std/SecureRandomCompanionPlatform.scala

```diff
@@ -110,4 +114,10 @@ private[std] trait SecureRandomCompanionPlatform {
     }
   }
 
+  def javaSecuritySecureRandom[F[_]: Sync]: F[SecureRandom[F]] =
+    Sync[F].delay(unsafeJavaSecuritySecureRandom)
+
+  private[effect] def unsafeJavaSecuritySecureRandom[F[_]: Sync]: SecureRandom[F] =
```

Comment:
It's convention for unsafe, side-effecting methods to always use parentheses to emphasize that they are side-effecting.

```suggestion
  private[effect] def unsafeJavaSecuritySecureRandom[F[_]: Sync](): SecureRandom[F] =
```